src/visudo/mod.rs

```diff
@@ -158,3 +159,11 @@ fn solve_editor_path() -> io::Result<PathBuf> {
         "cannot find text editor",
     ))
 }
+
+fn generate_tmp_path() -> PathBuf {
```

Comment:
I'd prefer if this read some random bytes from `/dev/urandom` to generate the name.

I think as is, this can run into name collisions when `visudo -f a` and `visudo -f b` are executed in parallel -- as each process is editing a different file a file lock won't prevent them from running in parallel, right?. we don't know the resolution of the monotonic timer so `Instant::now` could return the same value for several milliseconds -- or it could always return the same value on virtualized / emulated hardware.